docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -174,6 +174,19 @@ The following command verifies if alerts in an Alertmanager cluster are deduplic
 mimirtool alerts verify
 ```
 
+#### Render template
+
+You can render your alertmanager template by using the subcommand `render`.
+
+The following command render a template and prints it to the terminal.
+
+```bash
+# assuming we have written the templates out to the folder templates
+mimirtool alertmanager render --template-glob "templates/*" --template-data "alert_data1.json" --template-text '{{ template "my_message" . }}'
+```
+
+It can also take a glob path that will be expended.
```

Comment:
Given my other suggestion to move the options description before the comand example, I believe this can be removed.

```suggestion
```

Reply:
Modification made. Is that okay for you ?